apps/reference/docs/architecture.mdx

```diff
@@ -15,3 +15,25 @@ Supabase is not a 1-to-1 mapping of Firebase. While we are building many of the
 Our technological choices are quite different from Firebase. Everything we use is open source. Wherever possible, we use and support existing tools rather than developing from scratch.
 
 Most notably, we use Postgres rather than a NoSQL store. This choice was deliberate. We believe that no other database offers the scalability and functionality required to compete with Firebase.
+
+## Feature Status
+
+| Product                    | Feature                | Stage |
+| -------------------------- | ---------------------- | ----- |
+| Database                   | Postgres               | GA    |
+| Database                   | Webhooks               | Alpha |
+| Database                   | Point in time Recovery | Alpha |
+| Realtime                   | Postgres Changes       | Beta  |
+| Realtime                   | Broadcast              | Alpha |
+| Realtime                   | Presence               | Alpha |
+| Storage                    |                        | Beta  |
+| Storage                    | CDN                    | Beta  |
+| Edge Functions             |                        | Beta  |
+| Auth                       | OAuth Providers        | Beta  |
+| Auth                       | Passwordless           | Beta  |
+| Auth                       | Next.js Auth Helpers   | Alpha |
+| Auth                       | SvelteKit Auth Helpers | Alpha |
+| Public API                 |                        | Beta  |
+| CLI                        |                        | Beta  |
+| Client Library: JavaScript |                        | Beta  |
```

Comment:
Should this lib be considered Beta? It's pretty stable, even before v2

Reply:
updated to GA